src/App.js

```diff
@@ -150,6 +152,50 @@ export default class App extends Component {
       })
   }
 
+  updateClass (klass) {
+    let newClass
+    if (this.state.activeCharacter.hasOwnProperty('klass')) {
```

Comment:
Do you really need this `if` guard? Doesn't this also work?

```
Object.assign({}, this.state.activeCharacter, {klass})
```
regardless of whether `this.state.activeCharacter` has a `klass` attribute?